plugins/utils/todo-comments.nix

```diff
@@ -431,11 +430,10 @@ lib.nixvim.neovim-plugin.mkNeovimPlugin config {
     extraPackages = [ cfg.ripgrepPackage ];
 
     keymaps = lib.pipe cfg.keymaps [
-      (filterAttrs (n: v: v != null))
+      (filterAttrs (n: keymap: keymap != null))
       (mapAttrsToList (
         name: keymap: {
-          inherit (keymap) key options;
-          mode = "n";
+          inherit (keymap) key mode options;
           action =
             let
               cmd = keymapsActions.${name};
```

Comment:
```suggestion
              cmd = keymap.action;
```

Reply:
Actually, this would break the keymap, no? We construct a new action from the combination fo the option name and extraOptions.

Reply:
Seems like it's getting thrown for a loop because we didn't define action as something the user could choose before. So now, we want to inherit action unless cwd or keywords are defined. Unless we can inject those strings before <cr> conditionally.

Reply:
We don't need to have an `action` sub-option (we can pass `action=false` to `mkMapOptionSubmodule`), and it is a little pointless IMO, since it's just the same thing as the attrname tbh.

But I think the confusion here is that you're thinking of the plugin-keymap-option's `action` as the same as the _real_ (final) `action` to be used in the top-level `keymaps` option.